src/mumble/Settings.cpp

```diff
@@ -522,6 +522,7 @@ Settings::Settings() {
 	iTalkingUI_PrefixCharCount          = 3;
 	iTalkingUI_PostfixCharCount         = 2;
 	qsTalkingUI_AbbreviationReplacement = QLatin1String("...");
+	qsTalkingUI_BackgroundColor         = QLatin1String("#000000"); //could be set to QColor
```

Comment:
I think this should be a `QColor` in order to make the semantics of this field clearer. Also using a string here has the problem that it can potentially contain some contents that is not a valid color, so you'd have to perform some sort of explicit error checking